setup.py

```diff
@@ -54,6 +54,8 @@ def read(*names, **kwargs):
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: 3.12",
         "Topic :: Scientific/Engineering :: Bio-Informatics",
```

Comment:
Any reason that 3.13 and 3.14 are not included?  And that the no longer supported 3.8 and 3.9 are still supported?